template-overlay.nix

```
@@ -0,0 +1,51 @@
+## File initially generated by createOverlay
+## and then supposedly modified manually.
+## Some hints for manual modifications are in the file,
+## but the full doc is on nixos / nix packages website:
+## https://nixos.org/manual/nixpkgs/stable/#sec-language-coq
+
+{ lib, mkCoqDerivation, which, coq
+  ## declare extra dependencies here, to be used in propagateBuildInputs e.g.
+  # , mathcomp, coq-elpi
+  , version ? null, shim ? false }:
```

What is this `shim`? This is the first time I see this, it's not used in this derivation and it's not mentioned in the nixpkgs manual.

---

Woops, it's a HB specific thing I modeled from and accidentally let in.
```suggestion
  , version ? null }:
```